From 6aa11c2e96499a4a620b54433fd064961d1cbf66 Mon Sep 17 00:00:00 2001
From: Phillip Susi <psusi@ubuntu.com>
Date: Fri, 18 Apr 2014 11:01:03 -0400
Subject: [PATCH 101/103] libparted: remove last_usable_if_grown

Commit 3398e82a: "libparted: Use common function to calculate PTE sectors"
removed usage of the last_usable_if_grown variable, resulting in an error
because it is now unused but still defined.
---
 libparted/labels/gpt.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libparted/labels/gpt.c b/libparted/labels/gpt.c
index e1113ac..b7291a8 100644
--- a/libparted/labels/gpt.c
+++ b/libparted/labels/gpt.c
@@ -723,7 +723,7 @@ _parse_header (PedDisk *disk, const GuidPartitionTableHeader_t *gpt,
   GPTDiskData *gpt_disk_data = disk->disk_specific;
   PedSector first_usable;
   PedSector last_usable;
-  PedSector last_usable_if_grown, last_usable_min_default;
+  PedSector last_usable_if_grown;
   static int asked_already;
 
 #ifndef DISCOVER_ONLY
-- 
1.9.0

